scripts/build/debug/300-gdb.sh

```diff
@@ -230,6 +230,7 @@ do_gdb_backend()
     if [ "${static}" = "y" ]; then
         cflags+=" -static"
         ldflags+=" -static"
+        extra_config+=("--without-source-highlight")
```

Comment:
I don't think this will work. The option is `--enable-source-highlight` so what we want is `--enable-source-highlight=no` or `--disable-source-highlight`. I think the configure script might even do the right thing if we point PKG_CONFIG_LIBDIR at the right place.

Reply:
Fixed in 7ed7a6e